fork/dynamic-cdn-webpack-plugin/src/find.js

```diff
@@ -34,14 +34,15 @@ function findPackagesFromScopeFolder(scope, name, scopeFolderPath) {
 				// just add the path to the found list
 				return accu.concat(subFolderPath);
 			}
-			// the scope or package name is not the one we look for
-			// if there is a nested node modules folder, we dive into it for the search
-			const nestedNodeModulesPath = path.join(subFolderPath, 'node_modules');
-			if (fs.existsSync(nestedNodeModulesPath)) {
-				return accu.concat(
-					findPackagesFromNonScopeFolder(scope, name, nestedNodeModulesPath, []),
-				);
-			}
+			// NOT COMPATIBLE WITH PNPM
```

Comment:
This is not compatible with PNPM, it should be ok to go without it

Reply:
Note: Not sure of the impact of this change

Reply:
This is a fix, we can't merge that. It was the only way to find all versions of a given modules.
For example if you have prop-types twice and one of them can not be hoisted.